crates/chia-sdk-client/src/client.rs

```diff
@@ -0,0 +1,124 @@
+use std::{
+    collections::{HashMap, HashSet},
+    net::{IpAddr, SocketAddr},
+    ops::Deref,
+    sync::Arc,
+    time::{SystemTime, UNIX_EPOCH},
+};
+
+use chia_protocol::Message;
+use native_tls::TlsConnector;
+use tokio::sync::{mpsc, Mutex};
+
+use crate::{connect_peer, ClientError, Network, Peer};
+
+#[derive(Debug, Clone)]
+pub struct Client {
+    network_id: String,
```

Comment:
should we make this an enum? `mainnet` or `testnet``, right?

Reply:
You can pass in `simulator0` or a custom network technically